### DISCUSSION TITLE
LARGE AMOUNT OF DATA

### DISCUSSION THREAD

**Discussion by garder500** [BrainJS/brain.js#858] [Q&A]
I have about 300 000 datas of input/output for training a chatbot AI, is trainjs good for that ? Or should i use something else ?

It look it take very very much time for the first iteration (and for a forEach on each sentence it has take like an Hour) 

So... I want to know what do you think of it.

Thank's in advance

**Comment by pawanpratapweb** [comment]
you can use github workflow to automate the training.

i have done that in this repo: 
[https://github.com/pawanpratapweb/AI](https://github.com/pawanpratapweb/AI)
